boot/jaxrs/src/main/java/org/commonjava/indy/boot/jaxrs/JaxRsBooter.java

```diff
@@ -72,6 +74,27 @@ public static void main( final String[] args )
         }
     }
 
+    private static void setDefaultUncaughtExceptionHandler()
+    {
+        Thread.setDefaultUncaughtExceptionHandler( ( thread, error ) -> {
+            if ( error instanceof InvocationTargetException )
+            {
+                final InvocationTargetException ite = (InvocationTargetException) error;
+                System.err.println( "In: " + thread.getName() + "(" + thread.getId()
+                                                    + "), caught InvocationTargetException:" );
+                ite.getTargetException().printStackTrace();
+
+                System.err.println( "...via:" );
+                error.printStackTrace();
+            }
+            else
+            {
+                System.err.println( "In: " + thread.getName() + "(" + thread.getId() + ") Uncaught error:" );
+                error.printStackTrace();
+            }
+        } );
+    }
+
```

Comment:
Seems this handler is a duplicate one in propulsor? Maybe we should extract it to a common one in propulsor and reuse it here.

Reply:
They are different. Propulsor use 'Thread.currentThread().setUncaughtExceptionHandler'. The 'currentThread' make things not work. We need to update it later too.

Reply:
Pushed a Propulsor PR too.

Reply:
I don't mean the thread.setxxx, I mean the handler code itself(lambda). We can extract it to a common lambda in propulsor and use it directly here.

Reply:
With the propulsor pr, we can reuse the whole method. But we might want to do it next time to simplify the 2.0 release. Or we have to release Propulsor...